EIPS/eip-131.md

```diff
@@ -0,0 +1,85 @@
+---
+eip: 131
+title: Add precompiled BLAKE2b contract
+author: Tjaden Hess (@tjade273), Jay Graber, (@carterpy), James Hancock (@madeoftin)
+discussions-to: https://ethereum-magicians.org/t/blake2b-f-precompile/3157
+status: Draft
+type: Standards Track
+category: Core
+created: 2016-06-30
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+Special Projects wishes greater Ethereum interopability with Zcash, IPFS, and Handshake. Coincidentally, all of these projects use the Blake2 hash function. So, this is a request to add the Blake2b F precompile 11 to the Istanbul hardfork. The first special project here will probably be creating a wrapped ZEC (WZEC) within Ethereum as well as wrapped Ether within Zcash. After that, some yet-to-be-determined bridge architecture will allow Ethereum to benefit from Zcash’s shielded transactions.
```

Comment:
I am not fully sure this summary reflects this EIP. This EIP will not cause ZEC to be implemented. This block seems to be make more sense as part of the "Motivation" section.